content/introduction-and-setup/walkthrough/cloning-a-project/index.md

```diff
@@ -0,0 +1,104 @@
+
+Follow along with the provided walkthroughs to learn how to clone repos into Visual Studio. Try it out on your own by cloning the [LaunchCodeEducation/csharp-web-dev-datatypes](https://github.com/LaunchCodeEducation/csharp-web-dev-datatypes) repository page and fork the repository into your own GitHub account by selecting Fork from the top right of the page.
+
+## Windows Users
+
+[Clone a Git Repository in Visual Studio](https://learn.microsoft.com/en-us/visualstudio/version-control/git-clone-repository?view=vs-2022)
+
+Note the path where you save this repo.
+
+## Mac Users
+
+[Cloning an Existing Repo](https://learn.microsoft.com/en-us/visualstudio/mac/set-up-git-repository?view=vsmac-2022#clone-an-existing-repository)
+
+Be sure to note the Target Folder where this repo is saved.
+
+## Exploring the Cloned Repo in Your Terminal
+Once you have cloned it into Visual Studio, locate the repo using your terminal.
+
+On a Windows machine, it is a path you saved your repo at. On a Mac, it’s in the Target Folder.
+
+Once you find the repo, `cd` into it. Look for the Solution file which uses the `.sln` file type.
+
+```bash-session
+students-computer:csharp-web-dev-datatypes student$ ls
+HelloMethods                 csharp-web-dev-datatypes.sln
+TempConverter
+```
+
+{{% notice orange "Note" "rocket" %}}
+The `csharp-web-dev-datatype` solution contains 2 separate projects: `HelloMethods` and `TempConverter`. A single solution can hold multiple projects.
+{{% /notice %}}
+
+Try to open the solution using the command line prompt:
+
+Windows Users: `start *.sln`
+
+Mac Users: `open *.sln`
+
+Return to the terminal. Locate your `Program.cs` file for the `HelloMethods` project. This will be contained in the project directory of the same name.
+
+```bash-session
+students-computer:csharp-web-dev-datatypes student$ ls
+HelloMethods                 csharp-web-dev-datatypes.sln
+TempConverter
+students-computer:csharp-web-dev-datatypes student$ cd HelloMethods
+students-computer:csharp-web-dev-datatypes student$ ls
+HelloMethods.csproj  Message.cs              Program.cs
+```
+
+You have now stepped into the project files for `HelloMethods`. All of the files here are related to the `HelloMethods` project.
+
+As you work with repos in this unit, some solutions may contain a single project and others may contain multiple.
+
+## How to Work with a Cloned Repo
+
+We recommend using the terminal to open and work with your repos. You will be able to interact with git easily this way.
+
+Use the terminal to locate the repo you wish to open. `cd` into the solution. You can verify this by looking for a file that has the `.sln` type. Use the command prompt above for your operating system.
+
+Open the solution file tree in Visual Studio. If you see multiple projects, you can select which one to run two ways:
+
+1. Right-clicking on the name of the project and selecting the *Run* option.
+1. Open the project’s `Program.cs` file then use the run button in the menu bar.
```

Comment:
Why is this section necessary?  Students can just use the terminal in VS which automatically opens to the correct location.  What is `open *.sln` actually do?  Tried in both VS Terminal and my mac Terminal and nothing.  While it is nice to practice terminal commands this could be simplified.  Also can just use the Git menu as well.

Reply:
Added in an explanation about why we keep practicing those terminal skills! They are super important in demonstrating job readiness in many places. That command does work on my machine, but Visual Studio takes a while to start up and you may have to click on the icon to get it fired up. Added a note about this as well.